optax/_src/alias.py

```diff
@@ -431,6 +431,29 @@ def noisy_sgd(
   )
 
 
+def online_newton_step(learning_rate: ScalarOrSchedule,
+                       eps: float) -> base.GradientTransformation:
+  # pylint: disable=line-too-long
```

Comment:
Actually, I think as long as the link can fit into a line we could just introduce line breaks in the references. At least in the new theme e.g. for [rmsprop here](https://optax.readthedocs.io/en/latest/api.html#optax.rmsprop) I don't think there is a benefit to disabling and having it in one line.